examples/cc3220sf/xively_demo/main_tirtos.c

```diff
@@ -49,61 +49,67 @@
 /* Example/Board Header files */
```

Comment:
I haven't been pointing the code formatter at the examples that are based on sources from 3rd parties.  Instead I've been formatting the examples in a similar fashion to the other code that we lift from their SDK in the examples.  

But it's up to you. :)